README.md

```diff
@@ -88,10 +88,19 @@ SPI Vault instance has to be manually initialized. There is a script to help wit
 2) Clone SPI operator repo `git clone https://github.com/redhat-appstudio/service-provider-integration-operator && cd service-provider-integration-operator`
 3) run `vault-init.sh` script from repo root directory `./hack/vault-init.sh`
 
+### Optional: Use SharedSecret with Tekton Chains
 
-#### Post-bootstrap Tekton Chains Configuration
+During the build pipeline, it is possible to use the `redhat-appstudio-user-workload`
```

Comment:
This is no longer valid. Live/hot loading works now with the initContainer approach.

Reply:
Updated.